src/SFA.DAS.ApprenticeCommitments/Application/Commands/CreateApprenticeship/CreateApprenticeshipCommandHandler.cs

```diff
@@ -66,14 +66,11 @@ public class CreateApprenticeshipCommandHandler : IRequestHandler<CreateApprenti
             // return parameters for the invitation
             var res = new CreateApprenticeshipResponse
             {
-                ClientId = _loginConfiguration.IdentityServerClientId,
                 SourceId = id,
                 Email = apprentice.Email,
                 GivenName = apprentice.FirstName,
                 FamilyName = apprentice.LastName,
                 ApprenticeshipName = apprentice.CourseName,
-                CallbackUrl = _loginConfiguration.CallbackUrl,
```

Comment:
Are these three `_loginConfiguration` properties used anywhere else?  If not then I suggest removing them.